docs/redirects.config.js

```diff
@@ -259,7 +259,7 @@ const customRedirects = [
   },
   {
     source:
-      "/guides/:slug(waffle-testing|truffle-testing|truffle-migration|ganache-tests|using-custom-solc)",
+      "/guides/:slug(waffle-testing|truffle-testing|truffle-migration|ganache-tests)",
```

Comment:
@fvictorio I added the redirect because the link to the guide in the [Multiple solidity version guide](https://hardhat-git-fork-lilpolymath-docs-custom-solc-nomic-foundation.vercel.app/hardhat-runner/docs/advanced/multiple-solidity-versions) would be broken otherwise.

Reply:
Ah, right. But we should use the proper URL in that link, shouldn't we?

Reply:
Yes, fixed.